app/views/users/delete/show.html.slim

```diff
@@ -1,21 +1,17 @@
-= content_for :body_class do
-  ' modal-open
+.p0.cntnr-xxskinny.border-box.bg-white.rounded-xxl.modal-warning
+  h2.my2.fs-20p.sans-serif.regular.center
+    = t('users.delete.heading')
+  hr.mb3.bw4.rounded
+  .mb1.bold
+    = t('users.delete.subheading')
 
-= render layout: '/shared/modal_layout' do
-  .p4.cntnr-xxskinny.border-box.bg-white.rounded-xxl.modal-warning
```

Comment:
Due to the indentation changes, it's a bit hard to see exactly what changed. Am I correct thinking the main thing was a swap of `.p4` to `.p0` to tighten up the padding?

Reply:
yes

Reply:
yes